src/utils.ts

```diff
@@ -19,3 +19,8 @@ export const getVariance = (samples: number[], mean: number) => {
   const result = samples.reduce((sum, n) => sum + ((n - mean) ** 2));
   return result / (samples.length - 1) || 0;
 };
+
+/**
+ * Computes the sum of a sample
+ */
+export const getSum = (samples: number[]) => samples.reduce((sum, n) => sum + n, 0);
```

Comment:
Isn't totalTime so much faster than this?

Reply:
But the `getSum` will get a exact cost time (not include math calculate, array push or condition check operations).
Maybe you want sum the `totalTime` in every loop's end instead?

Reply:
Now I sum the `totalTime` in every loop's end